### PR TITLE
 🦋 Add (optional) be-BOP logo on PoS QR code #1614

### PR DESCRIPTION
**Pull request by ithiame**

### CODE REVIEW

File: src/lib/assets/logo-bebop.png
Comment (coyotte508) [inline src/lib/assets/logo-bebop.png:1]: Is it possible to have this in SVG instead?

File: src/routes/(app)/pos/session/+page.svelte [hunk 1]
```diff
@@ -157,6 +158,9 @@
 					alt="QR code"
 					class="h-96 w-96"
 				/>
+				{#if !data.removeBebopLogoPOS}
+					<img src={BEBOP_LOGO} alt="Logo" class="logo" />
```
Comment (coyotte508) [inline src/routes/(app)/pos/session/+page.svelte:162]:
```suggestion
					<img src={BEBOP_LOGO} alt="" class="logo" />
```

File: src/routes/(app)/pos/session/+page.svelte [hunk 2]
```diff
@@ -288,3 +292,17 @@
...
+		background-color: white;
+		position: absolute;
+		top: 30%;
+		left: 50%;
+		transform: translate(-50%, -50%);
+		width: 90px;
+		height: 90px;
+		object-fit: contain;
+		border-radius: 10px;
+	}
```
Comment (coyotte508) [inline src/routes/(app)/pos/session/+page.svelte:307]: Use tailwind classes directly on the logo instead

File: src/lib/assets/logo-bebop.png
Comment (Tirodem) [reply src/lib/assets/logo-bebop.png:1]: bebop-b
I hope github won't do a svg to png conversion again

Comment (Tirodem) [reply src/lib/assets/logo-bebop.png:1]: Grmpf.
[bebop-b.zip](https://github.com/user-attachments/files/18018040/bebop-b.zip)

**Review by coyotte508 — APPROVED** [review]

**Comment by Tirodem** [comment]
image
image
While QR code is readable, display is not good (depending of the PoS display device size & resolution) ❌

**Comment by coyotte508** [comment]
Indeed 

The logo should directly be integrated in the SVG - please include screenshots in your future PRs

File: src/lib/assets/bebop-b.svg
Comment (coyotte508) [inline src/lib/assets/bebop-b.svg:1]: The SVG contains a lot of useless metadata: 

```
<svg
   id="oo"
   viewBox="0 0 370.82057 561.73657"
   version="1.1"
   sodipodi:docname="bebop-b.svg"
   inkscape:version="1.1.2 (b8e25be833, 2022-02-05)"
   width="370.82059"
   height="561.73657"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview915"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="0.20831121"
     inkscape:cx="1024.9088"
     inkscape:cy="1459.355"
     inkscape:window-width="1920"
     inkscape:window-height="1017"
     inkscape:window-x="-8"
     inkscape:window-y="-8"
     inkscape:window-maximized="1"
     inkscape:current-layer="oo"
     fit-margin-top="0"
     fit-margin-left="0"
     fit-margin-right="0"
     fit-margin-bottom="0" />
```

**Comment by coyotte508** [comment]
Done 

image

Note that you can edit the `bebop-b.svg` if you want it to be larger (so whitespace is added on the sides)

When you edit it, make sure to remove the top-level fields (`<xml`, `xmlns:...`)

File: src/lib/assets/bebop-b.svg
Comment (Tirodem) [reply src/lib/assets/bebop-b.svg:1]: Made quickly with Inkscape from the main be-BOP svg logo, my bad, we'll upload a cleaner SVG later 🙊

**Review by Tirodem — APPROVED** [review]
Functional tests are OK ✅

**Pull request merged**